commons/src/main/java/com/powsybl/commons/reporter/LoggerReporter.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.commons.reporter;
+
+import org.apache.commons.text.StringSubstitutor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.function.Consumer;
+
+/**
+ * @author Florian Dupuy <florian.dupuy at rte-france.com>
+ */
+public class LoggerReporter extends AbstractReporter {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(LoggerReporter.class);
+
+    private static final String DEFAULT_ROOT_TASK_KEY = "rootTaskKey";
+    private static final Map<String, Consumer<String>> LOG_CONSUMER_MAP = Map.of(
+        Report.SEVERITY_TRACE, LOGGER::trace,
+        Report.SEVERITY_DEBUG, LOGGER::debug,
+        Report.SEVERITY_INFO, LOGGER::info,
+        Report.SEVERITY_WARN, LOGGER::warn,
+        Report.SEVERITY_ERROR, LOGGER::error
+    );
+
+    protected final String taskKey;
+    protected final String defaultName;
+    protected final Map<String, Object> taskValues;
+
+    public LoggerReporter() {
+        this(DEFAULT_ROOT_TASK_KEY, "Root task", Collections.emptyMap());
+    }
+
+    public LoggerReporter(String rootTaskKey, String rootDefaultName, Map<String, Object> taskValues) {
+        this.taskKey = Objects.requireNonNull(rootTaskKey);
+        this.defaultName = rootDefaultName;
+        this.taskValues = new HashMap<>(Objects.requireNonNull(taskValues));
+    }
+
+    @Override
+    public LoggerReporter createChild(String taskKey, String defaultName, Map<String, Object> values) {
+        getDefaultLogConsumer().accept(toString());
```

Comment:
is it ok to have a side effect here? What if you want to create a child, but not have the Logger.info log right now ? Also what if you want a different level than INFO for this log ?

Reply:
Good catch, `getDefaultLogConsumer()` is not appropriate here. I'll add a method returning a `Consumer<String>` which could be overriden, if you're ok with that. Looking for severity in task values would be consistent with the reports but severity of a task doesn't really make sense.

Reply:
Or just deleting that logging could be a solution in fact; not very useful information.